docs/3.0.0-beta.x/guides/auth-request.md

```diff
@@ -0,0 +1,172 @@
+# Authenticated request
+
+In this guide we will see how you can request the API as an authenticated user.
+
+## Introduction
+
+To let you understand many concepts mainly on the [roles and permissions](../plugins/users-permissions.md#manage-roles-permissions) part, we will user many users and roles.
```

Comment:
`To show you many of the concepts on the [roles and permissions](../plugins/users-permissions.md#manage-roles-permissions) part, we will use many users and roles.`